Describe identifiers more clearly in help and documentation (#128)
jenkinsci/pipeline-input-step-plugin: README.md

  input 'Proceed or Abort'

> Input Example with `Custom id`:
> Every input step has an unique id. It is used in the generated URL to proceed or abort.

Reviewer: ```suggestion
Input Example with `Custom identifier`:
Every input step has an unique identifier. It is used in the generated URL to proceed or abort.
```

> A specific id could be used, for example, to mechanically respond to the input from some external process/tool.

Reviewer: ```suggestion
A specific identifier could be used, for example, to mechanically respond to the input from some external process/tool.
```

   input message: 'input-message', ok: 'OK'

Input Example with `Allowed Submitter`:

> User ids and/or external group names of person or people permitted to respond to the input, separated by ','. Spaces will be trimmed automatically, so "Alice, Bob, Charles" is the same as "Alice,Bob,Charles".

Reviewer: In this case, I think that many places in the Jenkins documentation and changelogs use the word "username" to refere to the identifier of a user.
```suggestion
Usernames and/or external group names of those permitted to respond to the input, separated by ','. Spaces will be trimmed automatically, so "Alice, Bob, Charles" is the same as "Alice,Bob,Charles".
```

Author: done

Note: Jenkins administrators are able to respond to the input regardless of the value of this parameter.

   input message: '', submitter: 'jenkins-user, jenkins-user2'

Input Example with Parameter to store the `Approving Submitter`:

> If specified, this is the name of the return value that will contain the id of the user that approves this input. The return value will be handled in a fashion similar to the parameters value.

Reviewer: Username rather than `id` (I believe, though I'm not thrilled with having two occurrences of `user` in the same sentence.
```suggestion
If specified, this is the name of the return value that will contain the username of the user that approves this input. The return value will be handled in a fashion similar to the parameters value.
```

Author: done
